.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,46 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Description
+<!--- Describe your changes in detail -->
+
+## Checklist:
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+- [ ] **I am adding a new extension to an existing language.**
+  - [ ] The new extension is used in hundreds of repositories on GitHub.com
```

Comment:
The word "new" can be axed, because it's technically impossible to add an old extension.

I might reword this to *"I am associating a language with a new file extension"*, since it's clearer English.